crawler.py

```diff
@@ -179,6 +182,14 @@ def run_all(driver):
 
 os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
 os.environ['MOZ_HEADLESS'] = '1'
-driver = webdriver.Firefox(firefox_binary='tools/nightly/firefox-bin')
+
+# Download the latest Firefox coverage build
+latest_cov_build()
```

Comment:
We probably shouldn't download it every time we run the crawler, but manually by running the script.